### PR TITLE
Clear admin table filter before adding new row

### PR DESCRIPTION
**Pull request by Martchus**
See https://progress.opensuse.org/issues/51719

### CODE REVIEW

File: assets/javascripts/admintable.js [hunk 1]
```diff
@@ -3,20 +3,34 @@ function updateTextArea(textArea) {
...
+
+    // scroll to the new row
+    $('html').animate({scrollTop: $(row.node()).offset().top}, 250);
+}
+
+function addAdminTableRow() {
+    var adminTable = window.adminTable;
+
+    // clear search
+    $('#test-suites_filter input').val('');
```
Comment (Amrysliu) [inline assets/javascripts/admintable.js]: The id of filter div in machine table is "machines_filter",  Does it need  the same treatment?

Comment (Martchus) [reply assets/javascripts/admintable.js]: You're right - I will fix it.

**Comment by okurz** [comment]
+1 except for missing test

**Comment by Martchus** [comment]
@okurz Since we're now requiring at least one approval before being able to merge it would be good if you used the review feature. The new/adjusted lines were actually covered by tests but I added some additional checks.

**Comment by okurz** [comment]
I don't see "+1" as an approval but only as "did not find anything wrong, like the idea". If you would have added test I would feel comfortable to give the approval :)

**Comment by okurz** [comment]
so there you go, the test code looks nice, thanks. However we have checks failed. Better retrigger until green then forcing merge and we overlooked something else

**Comment by okurz** [comment]
Actually two checks have failed https://travis-ci.org/os-autoinst/openQA/jobs/541625018#L1103 and tidy in https://travis-ci.org/os-autoinst/openQA/jobs/541625016#L833 . I suggest we fix this first before we waste further time retriggering tests :)

**Comment by Amrysliu** [comment]
when someone who wants to add test suites,  they usually use the search function to find a exists test suite and copy settings or something else from them. So they want to the search result stay in the table. As your modification, does the search result disappear?

**Comment by Martchus** [comment]
@Amrysliu I see the use-case. Unfortunately it is not well supported anymore since I have made the admin tables proper data tables. This PR does not make things worse but is also not an improvement.

To support the copying use-case I need to include the empty row in the search after all. If this is wanted I can add the following commit: https://github.com/Martchus/openQA/commit/admin_table_2

**Comment by Amrysliu** [comment]
@Martchus Maybe this is wanted. I asked colleagues in Migration Team, they want this function.

**Comment by Martchus** [comment]
Btw, it would look like this:

screenshot_20190605_130735

So you search for e.g. 'foo bar', click 'New ...' and end up with something like the screenshot above.

**Comment by Amrysliu** [comment]
@Martchus  I think the expression is good.

**Comment by Martchus** [comment]
It is likely best to merge this PR and create another PR for the other commit when it is ready (tests still need adaption). And like I said - this use-case is already broken with the table in its current state (because one would need to clear the filter manually).

**Pull request merged**